### PR TITLE
PS Script refactoring

### PR DESCRIPTION
**Pull request by juhlee-microsoft**
1. Replaced with PS standard cmdlet names
2. Block format
3. Replaced with 'Seconds' parameter

### CODE REVIEW

**Comment by juhlee-microsoft** [comment]
image

File: Libraries/IntegrationServiceLibrary.psm1 [hunk 1]
```diff
@@ -521,17 +518,13 @@ function Create-Controller{
 	# Check if the controller already exists.
 	#
 	$scsiCtrl = Get-VMScsiController -VMName $vmName -ComputerName $server
-	if ($scsiCtrl.Length -1 -ge $controllerID)
-	{
-	Write-LogInfo "SCSI controller already exists"
-	}
-	else
-	{
+	if ($scsiCtrl.Length -1 -ge $controllerID) {
```
Comment (LiliDeng) [inline Libraries/IntegrationServiceLibrary.psm1]: if (($scsiCtrl.Length - 1) -ge $controllerID)

File: Testscripts/Windows/DiskHotAddRemove-Serial.ps1 [hunk 2]
```diff
@@ -45,7 +45,7 @@ function Main {
                 Write-LogInfo "$count - Successfully attached an empty data disk of size $diskSizeinGB GB to VM"
             } else {
                 $testResult = $resultFail
-                throw "Fail to attach�an empty data disk of size $diskSizeinGB GB to VM"
+                throw "Fail to attach�an empty data disk of size $diskSizeinGB GB to VM"
```
Comment (LiliDeng) [inline Testscripts/Windows/DiskHotAddRemove-Serial.ps1]: Expected symbol?

File: Libraries/IntegrationServiceLibrary.psm1 (hunk 1 above)
Comment (juhlee-microsoft) [reply Libraries/IntegrationServiceLibrary.psm1]: fixed in 05932f86e8b954c3b1a541ffc483c96ca1036710

File: Testscripts/Windows/DiskHotAddRemove-Serial.ps1 (hunk 2 above)
Comment (juhlee-microsoft) [reply Testscripts/Windows/DiskHotAddRemove-Serial.ps1]: fixed in 05932f86e8b954c3b1a541ffc483c96ca1036710

**Comment by juhlee-microsoft** [comment]
@LiliDeng anything else?

**Review by LiliDeng — APPROVED** [review]
Seems no logic change

**Pull request merged**